src/tests/bulkConcepts/actions/index.test.js

```diff
@@ -15,18 +16,39 @@ import
 fetchCielConcepts,
 { addExistingBulkConcepts, addDictionaryReference, fetchConceptSources } from '../../../redux/actions/bulkConcepts';
 import cielConcepts from '../../__mocks__/concepts';
+import api from '../../../redux/api';
 
 const mockStore = configureStore([thunk]);
 jest.mock('react-notify-toast');
 
 describe('Test suite for source concepts actions', () => {
+  api.mappings.fetchFromPublicSources = jest.fn(() => ({ data: [] }));
+
   beforeEach(() => {
     moxios.install(instance);
   });
 
   afterEach(() => {
     moxios.uninstall(instance);
   });
+  it('dispatches an error on fetchConceptSources failure', () => {
```

Comment:
Is this following our convention?

Reply:
I have adjusted the description to better fit conventions. It is worth noting that all tests in this file follow the previous convention i.e something similar to `it(dispatches an error on fetchConceptSources failure)`. If the change is satisfactory, I will create a ticket to adjust the test descriptions in this file